rfc/0020-table-value-types/README.md

```diff
@@ -0,0 +1,197 @@
+<!--
+SPDX-FileCopyrightText: 2025 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+-->
+
+# RFC 0020: Value types as table schema <!-- TODO: adapt title -->
+
+| | |
+|---|---|
+| Feature Tag | `table-schema-value-types` |
+| Status | `DRAFT` |
+| Responsible | `jrentlez` |
+<!-- 
+  Status Overview:
+  - DRAFT: The RFC is not ready for a review and currently under change. Feel free to already ask for feedback on the structure and contents at this stage.
+  - DISCUSSION: The RFC is open for discussion. Usually, we open a PR to trigger discussions.
+  - ACCEPTED: The RFC was accepted. Create issues to prepare implementation of the RFC.
+  - REJECTED: The RFC was rejected. If another revision emerges, switch to status DRAFT.
+-->
+
+## Summary
+
+This RFC changes the way sheets are parsed into tables, using a multi property
+value type as the table schema definition.
+
+## Motivation
+
+- Allows users to define how a sheet is parsed into a table.
+- Allows users to reuse table schema definitions.
+- Remove special syntax regarding table definitions.
```

Comment:
Not a motivation; rather focus on what the current problem is.. Are we not able to express something, is the syntax not consistent, ...